sysl2/sysl/codegen.go

```diff
@@ -261,6 +276,22 @@ func GenerateCode(rootModel, model, rootTransform, transform, grammar, start str
 	return codeOutput
 }
 
+func GenerateCodeWithParams(rootModel, model, rootTransform, transform, grammar, start string, loglevel string,
```

Comment:
This function is called only by Codegen_test. In that case, it is better to define it in test instead of source.

Reply:
ok